camera.py

```diff
@@ -0,0 +1,26 @@
+import pygame.camera
+import pygame.event as pg_event
+
+resolution = (1280, 720)
+camera_path = "/dev/video0"
+
+
+def camera(camera_path):
+    pygame.init()
+    pygame.camera.init()
+
+    cam = pygame.camera.Camera(camera_path, resolution)
+    cam.start()
+    window = pygame.display.set_mode(resolution)
+    while True:
+        image = cam.get_image()
+
+        pygame.display.set_caption('video')
```

Comment:
Probably this can be named as «call-app» or something :)

You can also pass this as a parameter to make it adjustable.

Reply:
OK, i'll pass it as a parameter and rename the window.